src/pages/itemList/ItemList.js

```diff
@@ -1,7 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ProductCategory from './components/ProductCategory.js';
+import ListItem from './components/ListItem.js';
+import LitsFilter from './components/ListFilter.js';
+import './ItemList.scss';
 
 const ItemList = () => {
-  return <div>ddd</div>;
+  const [listItems, setListItems] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/LIST_ITEMS.json', {
```

Comment:
포트가 바뀔 수 있기 때문에 아래와 같이 작성해주는게 좋습니다!
```suggestion
    fetch('/data/LIST_ITEMS.json', {
```